packages/datahub-data/src/db/schema.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  index,
+  timestamp,
+  mysqlTable,
+  serial,
+  varchar,
+  text,
+  unique,
+  int,
+} from 'drizzle-orm/mysql-core';
+import {sql, relations} from 'drizzle-orm';
+import {createInsertSchema} from 'drizzle-zod';
+
+export const objectLists = mysqlTable(
+  'object_list',
```

Comment:
Thinking out load: the names `object_list` and `object_item` are a bit generic and vague - what is a 'list' and what is an 'item'? Would it be an option to express the function in the names, e.g. an 'object item' is a `bookmark` and the 'object list' is a collection of `bookmarks`?

Reply:
Decided in the meeting: We will keep the chosen name for now.